fresnel/geometry.py

```diff
@@ -183,6 +183,183 @@ def radius(self):
     def color(self):
         return util.array(self._geometry.getColorBuffer(), geom=self)
 
+
+class Box(Cylinder):
+    R""" Box geometry.
+
+    Generate a triclinic box shape.
+
+    Args:
+        scene (:py:class:`fresnel.Scene`): Add the geometry to this scene
+        box (`numpy.ndarray` or `array_like`): (``1x1, 1x3, or 1x6`` : ``float32``) Assumes 1x1 is cubic, 1x3 is orthorhombic, and 1x6 is triclinic.
```

Comment:
I'm not sure how fresnel documents things elsewhere, but to me `1x1` means `[[side_length]]` and `1x3` means `[[x, y, z]]`. Just be consistent with how fresnel describes 1D arrays elsewhere.

Reply:
Here is an example of how fresnel currently documents array sizes:
```
(`numpy.ndarray` or `array_like`): (``3``, ``4``, ``Nx3``, or ``Nx4`` : ``float32``) - ``RGB`` or ``RGBA``
```